### PR TITLE
[Feature][docs] Add the Issue and Pull Request Chinese and English docs

### PR DESCRIPTION
**Pull request by yangyichao-mango**
image
image

### CODE REVIEW

File: docs/zh-cn/development/pull-request.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <tbody>
+        <tr>
+            <td style="text-align: center;">Feature</td>
+            <td style="text-align: center;">Feature</td>
+            <td style="text-align: center;">[Feature-3333][server] Implement xxx</td>
+        </tr>
+        <tr>
+            <td style="text-align: center;">Bug</td>
+            <td style="text-align: center;">Hotfix</td>
+            <td style="text-align: center;">[Hotfix-3333][server] Fix xxx</td>
```
Comment (dailidong) [inline docs/zh-cn/development/pull-request.md:33]: I think "fix" is much better than Hotfix, there is not all bug is belong to Hotfix? how do you think ?

Comment (yangyichao-mango) [reply docs/zh-cn/development/pull-request.md:33]: I agree with you. I will change it.

Comment (yangyichao-mango) [reply docs/zh-cn/development/pull-request.md:33]: > I think "fix" is much better than Hotfix, there is not all bug is belong to Hotfix? how do you think ?

I agree with you. I will change it.

**Comment by samz406** [comment]
I think it can be explained in the pr description. If the newly developed function page is different from the previous one, you can simply provide a user manual to facilitate the user to get started quickly, or you can submit the user manual to the official website manual later.

我觉得可以在pr描述中说明一下，如果是新开发的功能页面和之前不一样可以简单提供一个使用手册，方便用户快速上手，或者可以将使用手册后续提交到官网手册上。

**Comment by yangyichao-mango** [comment]
> I think it can be explained in the pr description. If the newly developed function page is different from the previous one, you can simply provide a user manual to facilitate the user to get started quickly, or you can submit the user manual to the official website manual later.
> 
> 我觉得可以在pr描述中说明一下，如果是新开发的功能页面和之前不一样可以简单提供一个使用手册，方便用户快速上手，或者可以将使用手册后续提交到官网手册上。

I agree with you, and I will record that.

**Comment by yangyichao-mango** [comment]
Pending for the complete of #146 .

**Pull request merged**